internal/sdkv2provider/resource_user_identity_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package sdkv2
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func testAccResourceUserIdentity(t *testing.T) resource.TestCase {
+	return resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: configResourceBasicIdentity,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestMatchResourceAttr(
+						"corner_user_identity.foo", "name", regexp.MustCompile("^For")),
+				),
+				ExpectNonEmptyPlan: true,
```

Comment:
You can remove this field. `ExpectNonEmptyPlan` just modifies the default behavior of the testing framework to assert the plan has a diff after applying (uncommon, usually caused by drift/data consistency issues).

The default behavior of the test step will assert that the plan is empty after running an apply, which in this case, seems appropriate 👍🏻 

```suggestion
```